mpls-lab/extra/hosts_file.conf

```diff
@@ -0,0 +1,11 @@
+cae-1: 10.70.70.12
```

Comment:
It would be good to leave a comment that this hosts file needs to be inserted on the GNS3 GUI configure pane of the device

Reply:
Added the description in the comments of the file.